main.go

```diff
@@ -81,6 +80,7 @@ func main() {
 	if err = (&controllers.HASApplicationReconciler{
 		Client: mgr.GetClient(),
 		Scheme: mgr.GetScheme(),
+		Log:    ctrl.Log.WithName("controllers").WithName("HASApplication"),
```

Comment:
@maysunfaisal So, initially I did commit your suggestion, but thinking about it further, I think we should use a separate name (i.e. `HASApplication`, `HASComponent`, etc rather than `application-service`) for each controller's logger. 

My thinking is that it'll be easier to differentiate which controller in the operator the logs are coming from.

Reply:
right that makes sense, i missed that it was the HASApp Reconciler, nvm then